src/styles/blocks/header.scss

```diff
@@ -0,0 +1,75 @@
+@use '../utils/variables' as *;
+
+.header {
+  font-family: $font-family;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), for colors, it is recommended to use variables. Consider using a variable for the box-shadow color if this color is reused in other places.